src/DynamicListForm.jsx

```diff
@@ -21,8 +21,13 @@ export class DynamicListForm extends React.Component {
 
     removeItem(idx, field, value) {
         this.setState(state => {
+            const validationFailedDelta = this.props.validationFailed ? [...this.props.validationFailed] : [];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230725-122305-68cb803a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)